app/ts/components/Blockie.tsx

```diff
@@ -0,0 +1,144 @@
+// ported directly from https://github.com/DarkFlorist/TheInterceptor
+import { useRef, useEffect } from 'preact/hooks'
+
+interface BlockieProps {
+	seed: string
+	size?: number
+	scale?: number
+	color?: string
+	bgColor?: string
+	spotColor?: string
+	borderRadius?: string
```

Comment:
```suggestion
	seed: string
	scale?: number
	borderRadius?: string
```
Even `borderRadius` could probably be dropped and just leave it up to HTML to deal with rather than dealing with it in the blockie.